python-sdk/src/astro/extractors/facets.py

```diff
@@ -0,0 +1,51 @@
+from typing import Dict, List, Optional
+
+import attr
+from openlineage.client.facet import BaseFacet
+
+from astro.constants import FileType
+from astro.table import Column, Metadata
+
+
+@attr.s
+class InputFileDatasetFacet(BaseFacet):
+    """
+    Facet that represents input dataset Facets for load file
+    :param file_size: size of the file.
+    :param number_of_files: number of files to be loaded.
+    :param type: type of the file.
+    :param description: description of the dataset.
+    :param is_pattern: True when file path is a pattern(eg. s3://bucket/folder or /folder/sample_* etc).
+    :param files: list of filepaths to be loaded from dataset.
+    """
+
+    file_size: Optional[int] = attr.ib()
+    number_of_files: Optional[int] = attr.ib()
+    file_type: FileType = attr.ib()
+    description: Optional[str] = attr.ib(default=None)
+    is_pattern: bool = attr.ib(default=False)
+    files: List[str] = attr.ib(default=[])
```

Comment:
If there are multiple files, what does `file_size` indicate over here?

Should we have multiple Facets for each file instead? thoughts?

Reply:
Yes created `InputFileFacet` in [this](https://github.com/astronomer/astro-sdk/pull/1025/commits/aa26fc665b0775941c9ceda3149ba205d94369dc). That makes better sense.